sdk/docker/README.md

```diff
@@ -0,0 +1,28 @@
+Blackbox Actions
+================
+
+1. Download and install the OpenWhisk CLI
+2. Install OpenWhisk Docker action skeleton.
+3. Add user code
+4. Build image
+5. Push image
+6. Test out action with CLI
+
+The script `buildAndPush.sh` is provided for your convenience. The following command sequence
+runs the included example Docker action container using OpenWhisk.
+
+```
+# install ./dockerSkeleton with example
+> wsk sdk install docker
+# change working directory
+> cd dockerSkeleton
```

Comment:
@sjfink I am in the process of moving per suggestions above. You got an early preview. 

Will rename clientApp.